base/broadcast.jl

```diff
@@ -96,7 +96,7 @@ dumpbitcache(Bc::Vector{UInt64}, bind::Int, C::Vector{Bool}) =
             # reverse-broadcast the indices
             @nexprs $nargs i->(I_i = newindex(I, imap_i))
             # extract array values
-            @nexprs $nargs i->(@inbounds val_i = _broadcast_getvals(A_i, I_i))
```

Comment:
Might be better to keep the `@inbounds` and then use `@propagate_inbounds` rather than calling `unsafe_getindex` above?